platypus/commands.py

```diff
@@ -0,0 +1,163 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, platypus development team.
+#
+# Distributed under the terms of the GPL License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from os.path import join
+from operator import itemgetter
+from click import BadParameter
+from skbio.util import create_dir
+
+from platypus.parse import (parse_first_database, parse_second_database,
+                            process_results)
+
+def compare(interest_fp, other_fp, output_dir='blast-results-compare',
+            interest_pcts=None, interest_alg_lens=None, other_pcts=None,
+            other_alg_lens=None, hits_to_first=False, hits_to_second=False):
+    """Compare two databases and write the outputs
+
+    Parameters
+    ----------
+    interest_fp : str
+        BLAST results when searching against the database of interest.
+    other_fp : str
+        BLAST results when searching against the other database.
+    output_dir : str, optional
+        Name of the output file path.
+    interest_pcts : list, optional
+        Minimum percentage identity to be considered as a valid result in the
+        interest database search results. If None is passed, it defaults to
+        `[70]`.
+    interest_alg_lens : list, optional
+        Minimum alginment length to be considered a valid result in the
+        interest database search results. If None is passed, it defaults to
+        `[50]`.
+    other_pcts : list, optional
+        Minimum percentage identity to be considered as a valid result in the
+        other database search results. If None is passed, it defaults to
+        `[70]`.
+    other_lengths : list, optional
+        Minimum alginment length to be considered a valid result in the
+        other database search results. If None is passed, it defaults to
+        `[50]`.
+    hits_to_first : bool, optional defaults to False
+        Outputs the labels and counts of the sequences being hit in the first
+        database.
+    hits_to_second : bool, optional defaults to False
+        Outputs the labels and counts of the sequences being hit in the second
+        database.
+
+    Raises
+    ------
+    click.BadParameter
+        If the `interest_pcts` and the `other_pcts` lists are of different
+        length.
+        If the `interest_alg_lens` and the `other_alg_lens` lists are of
+        different length.
+
+    Notes
+    -----
+    Antonio, can you think of some notes.
```

Comment:
No, do we need notes? ... If we do, what about:
Long live the Platypus, our magnanimous lord!